viewing_party/party.py

```diff
@@ -1,23 +1,139 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if title == None or genre == None or rating == None:

```

Comment:
Nice handling for returning `None` for missing data! In this case we would want to use `is` rather than `==` for comparison because we are checking for `None`.

When comparing things in python, `==` compares if the values are equivalent, while `is` checks if they have the same identity. The `==` operator for objects can also be overloaded (re-defined) while `None` is a "Singleton object", an object which only ever has one instance, and thus one ID. Since there is only one instance of `None` allocated by python, all variables holding None must be pointing to the same ID in memory. All this put together means:
- When comparing a variable against `== None` could give us the wrong value.
- We want to use `is`, because if a variable holds None, it always points to the same ID as the None singleton object we refer to when we type out `None`.

Something else we can consider: here we're explicitly checking if the values are None, but what if we got empty strings for one of the values? If we wanted to check for those at the same time as we check for None, we could rely on the truthy/falsy values of the parameters and write something like: if not title or not genre or not rating: